### ISSUE TITLE
Icons are cut off

### ISSUE THREAD

**Issue by jukos** [andreaordonselli/qgis2o.gis#9]
Hello,
in my example I have used QGIS svg icons, 8mm size. What could be the reason that the icons are cut off above a certain scale? See screenshot.

![Icons cut](https://github.com/user-attachments/assets/78367935-7373-44dd-99d9-d2529d71268d)

Greetings

QGIS 3.36.3
qgis2opengis 2.21

**Comment by andreaordonselli** [comment]
Many thanks for your report, I already know this problem and I plan to fix it in the next version. It happens because the layer extent is fixed while I have to find a way to dynamize it based on the zoom.
For now you can act in the layers.js  file by modifying the extent, After the definition of the layer you will find the calculation and you can change the value 200 by increasing it

example:
```
jsonSource_Idrografia.getExtent();
var buffer_extent_Idrografia = new ol.extent.buffer(extent_Idrografia, 200);
```

**Comment by andreaordonselli** [comment]
This issue has been resolved in the v2.22.0 release
Thanks for your report
